libs/langchain-core/src/messages/ai.ts

```diff
@@ -435,6 +451,22 @@ export class AIMessageChunk<
     };
   }
 
+  protected override _prettyPrintDetails(): string[] {
```

Comment:
This duplication with `_prettyPrintDetails` above is awkward. Seems to be due to the inheritance / interface hierarchy of `AIMessageChunk`. @hntrl any thoughts on this one? I could obv factor into a function but was hoping to understand the situation a bit better before blindly doing that

Reply:
Huh where were you thinking we'd hoist it to?